types/mdx/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "npm-naming": false,
+        "no-single-declare-module": false
+    }
```

Comment:
would it be possible to inline these in the places it is needed?

> The linter configuration file, tslint.json should contain { "extends": "dtslint/dt.json" }, and no additional rules.
>
> If for some reason some rule needs to be disabled, disable it for that specific line using // tslint:disable-next-line:[ruleName] — not for the whole package, so that disabling can be reviewed. (There are some legacy lint configs that have additional contents, but these should not happen in new work.)

https://github.com/DefinitelyTyped/DefinitelyTyped#linter-tslintjson

Reply:
The rule `npm-naming` is reported on the first line of `index.d.ts`. If this is disabled, it precedes the header, meaning the header can’t be parsed.

Since `npm-naming` is already disabled and `no-single-declare-module` is reported twice, I think it’s ok to disable that rule here too. Also if we decide to merge the `.md` module declaration into `index.d.ts`, this rule will no longer be supported, so let’s resolve that discussion first. :)